test/core/pipeline/test_pipeline_base.py

```diff
@@ -176,6 +176,13 @@ def test_draw(self, mock_to_mermaid_image, tmp_path):
         assert image_path.read_bytes() == mock_to_mermaid_image.return_value
 
     # UNIT
+    def test_add_invalid_component_name(self):
+        pipe = PipelineBase()
+        with pytest.raises(ValueError):
+            pipe.add_component("this.is.not.a.valida.name", FakeComponent)
+        with pytest.raises(ValueError):
+            pipe.add_component("debug", FakeComponent)
```

Comment:
Sorry a little confused if this is expected to fail? It seems like the check is for components named `_debug`